website/templates/emails/contributor_added_preprint_node_from_osf.txt.mako

```diff
@@ -0,0 +1,23 @@
+<%!
+    from website import settings
+%>
+
+Hello ${user.fullname},
+
+${referrer_name + ' has added you' if referrer_name else 'You have been added'} as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
+
+This project also has a public preprint, discoverable at: ${settings.DOMAIN.rstrip('/') + node.preprint_url}
```

Comment:
This should use the preprint's absolute URL: https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/preprint_service.py#L93 (branded domains)